includes/wf_crm_webform_postprocess.inc

```diff
@@ -1915,6 +1915,10 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
     $params['source'] = $this->settings['new_contact_source'];
     $params['item_name'] = $params['description'] = t('Webform Payment: @title', array('@title' => $this->node->title));
 
+    // We need the submitted billing params from the request (Eg. billing_country_id-5)
+    //  because that's how payment processors expect to find them.
+    $params = array_merge($params, $_REQUEST);
```

Comment:
Just a  hot take - looks like a potential security risk merging 'anything the user wants to input' into params

Reply:
I've changed it to:
```
    foreach ($this->form_state['input'] as $formKey => $formValue) {
      if (substr($formKey, 0, 8) === 'billing_') {
        $params[$formKey] = $formValue;
      }
    }
```
Is that better?